src/visualizations/AgeGroupsChart.fs

```diff
@@ -315,93 +315,94 @@ let renderChartOptions
-              allowDecimals = ChartMode.ScaleType state.ChartMode = Relative
-           |}
-       plotOptions = pojo
-           {| series = pojo
-               {| stacking = "normal" |}
-           |}
-       tooltip = pojo
-           {| formatter = fun () ->
-                let sex = jsThis?series?name
-                let ageGroup = jsThis?point?category
-                let dataValue: float = jsThis?point?y
-
-                match state.ChartMode with
-                | AbsoluteInfections ->
-                    sprintf
-                        "<b>%s</b><br/>Starost: %s<br/>Potrjeno okuženi: %A"
-                        sex
-                        ageGroup
-                        (abs dataValue)
-                | InfectionsPerPopulation ->
-                    sprintf
-                        "<b>%s</b><br/>Starost: %s<br/>Delež okuženega prebivalstva: %s<br/>Prebivalcev skupaj: %d"
-                        sex
-                        ageGroup
-                        (percentageValuesLabelFormatter dataValue)
-                        (populationOf sex ageGroup)
-                | AbsoluteDeaths ->
-                    sprintf
-                        "<b>%s</b><br/>Starost: %s<br/>Umrli: %A"
-                        sex
-                        ageGroup
-                        (abs dataValue)
-                | DeathsPerPopulation ->
-                    sprintf
-                        "<b>%s</b><br/>Starost: %s<br/>Delež umrlih med prebivalstvom: %s<br/>Prebivalcev skupaj: %d"
-                        sex
-                        ageGroup
-                        (percentageValuesLabelFormatter dataValue)
-                        (populationOf sex ageGroup)
-                | DeathsPerInfections ->
-                    sprintf
-                        "<b>%s</b><br/>Starost: %s<br/>Delež umrlih glede na št. okuženih: %s"
-                        sex
-                        ageGroup
-                        (percentageValuesLabelFormatter dataValue)
-           |}
-       series = [|
-           {| name = LabelMale
-              color = "#73CCD5"
-              dataLabels = pojo
-                {| enabled = true
-                   formatter = fun() -> valuesLabelFormatter jsThis?y
-                   align = "right"
-                   style = pojo {| textOutline = false |}
-                   padding = 10 |}
-              data = chartData.MaleValues
-                     |> Array.map (Option.map (fun y -> -y))
-               |}
-           {| name = LabelFemale
-              color = "#D99A91"
-              dataLabels = pojo
-                {| enabled = true
-                   formatter = fun () -> valuesLabelFormatter jsThis?y
-                   align = "left"
-                   style = pojo {| textOutline = false |}
-                   padding = 10 |}
-              data = chartData.FemaleValues
-               |}
-       |]
+    {| Highcharts.optionsWithOnLoadEvent "covid19-age-groups" with
```

Comment:
Does this line belong into this PR? (everything below was just re-indented)

Reply:
It does not belong directly but it does :-) to use the onLoad event trigger in the Map consistently across all visualizations. Indentation is important (anonymous record vs anonymous record extension).